project/src/components/comment-form/comment-form.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+
+import RadioButton from '../radio-button/radio-button.jsx';
+
+function CommentForm() {
+  const [formData, setFormData] = useState({
+    rating: '',
+    review: '',
+  });
+
+  const handleChange = (event) => {
+    const {name, value} = event.target;
+    setFormData({...formData, [name]: value});
+  };
+
+  const radioButtons = [
+    {value: '5', title: 'perfect'},
+    {value: '4', title: 'good'},
+    {value: '3', title: 'not bad'},
+    {value: '2', title: 'badly'},
+    {value: '1', title: 'terribly'},
+  ];
+
+  return (
+    <form className="reviews__form form" action="#" method="post">
+      <label className="reviews__label form__label" htmlFor="review">Your review</label>
+      <div className="reviews__rating-form form__rating">
+        {radioButtons.map((item) => <RadioButton key={item.value} value={item.value} title={item.title} checked={formData.rating === item.value} handleChange={handleChange} /> )}
```

Comment:
```suggestion
        {radioButtons.map((item) => (
        <RadioButton 
          key={item.value} 
          value={item.value} 
          title={item.title} 
          checked={formData.rating === item.value} 
          handleChange={handleChange}
         />
         ))}
```

форматирование поправь. Сделай более читабельным.